README.md

```diff
@@ -77,11 +77,30 @@ These values are publicly visible on GitHub and **should not be used in producti
 > :exclamation: <br> 
 > At the time of writing, upload within projects created in the development database will most likely not work. <br>
 > To use the upload functionality with the `CLI`, first create a project.
+
 ### Database changes
 
-If the database is modified, you will need to rebuild the containers from scratch.
+> :heavy_exclamation_mark: Do your database changes in `models.py` while the containers are running (`docker-compose up`). Do not restart them to regenerate the database.
+
+If you modify the database models (e.g. tables or indexes), you must create a migration for the changes. We use `Alembic` (via `flask-migrate`) which compares our database models with the running database to generate a suggested migration.
+
+Run the command `flask db migrate -m <commit message/name>` in the running backend:
+
+```bash
+docker exec dds_backend flask db migrate -m <commit message/name>
+```
 
-First, remove all docker containers and volumes
+This will create a migration in the folder `migrations/versions`. Confirm that the changes in the file match the changes you did, otherwise change the `upgrade` and `downgrade` functions as needed. Keep an eye out for changes to the `apscheduler` tables and indexes, and make sure they are not included in the migration. Once the migration looks ok, test it by running `flask db upgrade` in the backend:
```

Comment:
One could add here that it's perfectly fine to delete the new version generated and rerun the migrate command to generate a new file.

Reply:
The old migrations file also has to be restored to avoid the reference to the new file. I decided to word it as
> If you want to start over, restore the content of `migrations/versions` and start from autogeneration method 2.